tests/containers/bci_prepare.pm

```diff
@@ -75,13 +79,13 @@ sub run {
         }
         assert_script_run('pip3 --quiet install --upgrade pip', timeout => 600);
         assert_script_run("pip3 --quiet install tox", timeout => 600);
-    } elsif ($host_distri eq 'centos') {
+    } elsif ($host_distri eq 'centos' || $host_distri eq 'rhel') {
```

Comment:
```suggestion
    } elsif ($host_distri =~ m/centos|rhel/) {
```

Reply:
Agreed! I have thought of it, but was not sure which would be cleaner codewise. Good that you have raised it. I will change it. Thanks